Multisig/UI/App/UIKit/CollectiblesViewController/CollectiblesHeaderView.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  CollectiblesHeaderView.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 02.11.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class CollectiblesHeaderView: UITableViewHeaderFooterView {
+    @IBOutlet weak var headerImageView: UIImageView!
+    @IBOutlet weak var headerNameLabel: UILabel!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        backgroundView = UIView(frame: self.bounds)
+        backgroundView!.backgroundColor = .gnoSystemWhite
```

Comment:
I believe this could be done in the XIB? This would reduce this boilerplate code

Reply:
Also, I get this warning in the console: `Changing the background color of UITableViewHeaderFooterView is not supported. Use the background view configuration instead`

Reply:
done